src/core/agent/services/signifyNotificationService.test.ts

```diff
@@ -1028,3 +1071,280 @@ describe("Long running operation tracker", () => {
     expect(operationsGetMock).not.toBeCalled();
   });
 });
+
+describe("Long running operation tracker", () => {
```

Comment:
Not sure if you've fully merged develop in or not because these changes are still here.
`git fetch && git merge origin/develop`